### PR TITLE
Fixed missing downloads

### PR DESCRIPTION
**Pull request by AndreasPrang**
Fixed the missing downloads and broken dependencies.

### CODE REVIEW

File: 5.1.3/Dockerfile [hunk 1]
```diff
@@ -45,7 +46,7 @@ RUN cd /tmp && wget http://ftp.gnu.org/gnu/bison/bison-2.1.tar.gz \
   && make clean
 
 RUN apt-get install -y libfreetype6-dev
-RUN cd /tmp && wget http://iweb.dl.sourceforge.net/project/freetype/freetype2/2.6/freetype-2.6.tar.gz \
+RUN cd /tmp && wget http://download.savannah.gnu.org/releases/freetype/freetype-2.6.tar.gz \
```
Comment (ain) [inline 5.1.3/Dockerfile:49]: Thanks for a fix!

File: 5.1.3/httpd.conf [hunk 2]
```diff
@@ -2,13 +2,13 @@
 # This is the main Apache HTTP server configuration file.  It contains the
 # configuration directives that give the server its instructions.
 # See <URL:http://httpd.apache.org/docs/2.2> for detailed information.
-# In particular, see
+# In particular, see
```
Comment (ain) [inline 5.1.3/httpd.conf:5]: Let's not change `httpd.conf` as it'd break legacy apps that are based on this image. 

If anyone needs a change on this, they can mount in their own (or extend).

File: 5.1.3/Dockerfile [hunk 3]
```diff
@@ -86,6 +87,4 @@ COPY ./httpd.conf /usr/local/apache2/conf/httpd.conf
 
 COPY ./php.ini /usr/local/lib/php.ini
 
-RUN echo "FromLineOverride=YES" >> /etc/ssmtp/ssmtp.conf
-
-RUN ln -s /usr/local/apache2/htdocs /var/www
+RUN echo "FromLineOverride=YES" >> /etc/ssmtp/ssmtp.conf
```
Comment (ain) [inline 5.1.3/Dockerfile:90]: Same here, let's keep the symlink for legacy purposes. Please also retain EOL.

**Review by ain — CHANGES_REQUESTED** [review]

File: 5.1.3/httpd.conf (hunk 2 above)
Comment (AndreasPrang) [reply 5.1.3/httpd.conf:5]: When I tried to run the Image, the apache2 2.2.32 won't start with the old `http.conf`.

File: 5.1.3/Dockerfile (hunk 3 above)
Comment (AndreasPrang) [reply 5.1.3/Dockerfile:90]: OK. lets keep the symlink.

File: 5.1.3/httpd.conf (hunk 2 above)
Comment (ain) [reply 5.1.3/httpd.conf:5]: I see. Thanks for feedback on this, I'll take a second look into it and confirm.

**Comment by ain** [comment]
Closing. Dockerfile compilaton fails over `wheezy` dependencies. Complete revamp required.

**Pull request closed without merging**